src/state/slices/localWalletSlice/localWalletSlice.ts

```diff
@@ -6,22 +6,28 @@ export type LocalWalletState = {
   walletType: KeyManager | null
   walletDeviceId: string | null
   nativeWalletName: string | null
+  rdns: string | null
 }
 
 const initialState: LocalWalletState = {
   walletType: null,
   walletDeviceId: null,
   nativeWalletName: null,
+  rdns: null,
```

Comment:
Added to this slice since we persist it (but do not persist wallet context, as it is a context, not a slice). As soon as we connect a wallet, this gets set in the slice and persisted, so refreshes aren't borked.